lib/atomic/organizations.ex

```diff
@@ -222,4 +249,107 @@ defmodule Atomic.Organizations do
       :owner -> [:member, :admin, :owner]
     end
   end
+
+  @doc """
+  Returns the list of users organizations.
+
+  ## Examples
+
+      iex> list_users_organizations()
+      [%UserOrganization{}, ...]
+
+      iex> list_users_organizations()
+      [%UserOrganization{}, ...]
+
+  """
+  def list_users_organizations(opts \\ [], preloads \\ []) do
```

Comment:
```suggestion
  def list_users_organizations(opts \\ [], preloads \\ []) do
```
Make sure to update documentation in order with the function